### ISSUE TITLE
No diffs in remove entries

### ISSUE THREAD

**Issue by Thomas-Philippot** [DamienHarper/DoctrineAuditBundle#197]
Hello, I wonder why I don't have any diff on remove entries.
I have Diffs for insert and update but nothing but the entity id for remove entries.

Is that a choice ? Can I configure something to add the diff to the remove audit entry ?

**Comment by DamienHarper** [comment]
Hi @Thomas-Philippot you're right, remove operations do not show any diff data as long as this kind of operation is the result of a `DELETE ...` or `UPDATE ... SET deleted_at = ...` SQL query. 
What do you expect? Could you explain a bit more your use case?

**Comment by Thomas-Philippot** [comment]
Hello, let's assum my Entity has two attribute `id` and `name` When you `INSERT` you can see in the diffs, the two attribute `before` and `after`.

With before with null or empty values (as it was not existing before) and after with the values.

I Want to have the exact same behavior when performing a `Delete` (actually setting `deleted_at`), so I can see my attributes in `before`.
Hop I made myself clear enough.

**Comment by DamienHarper** [comment]
@Thomas-Philippot Thanks for the explanation, right now this is not possible. If I understand right, you expect to have all properties and their values logged in the audit entry. 
I mark this issue as a feature request to keep track of it.

**Comment by markus-fischbacher** [comment]
I would as well appreciate that feature.

**Comment by test789t** [comment]
Is there a development in the case?

**Comment by DamienHarper** [comment]
@test789t no, not that I know. BTW, I'd be glad to review a PR if you want to contribute.
